### PR TITLE
Thunderbird 60 compatibility

### PR DESCRIPTION
**Pull request by m-v-b**
Hello,

This is a quick-and-dirty commit to make the Thunderbird add-on compatible with Thunderbird 60.

Thank you!

### CODE REVIEW

**Comment by m-v-b** [comment]
Oops! I forgot to sign this commit. Will be done shortly.

File: chrome/content/qubesattachment.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ var qubesattachment = {
 
               var process = Components.classes["@mozilla.org/process/util;1"].createInstance(Components.interfaces.nsIProcess);
               var procPath = Components.classes["@mozilla.org/file/local;1"]
-                  .createInstance(Components.interfaces.nsILocalFile);
+                  .createInstance(Components.interfaces.nsIFile);
```
Comment (marmarek) [inline chrome/content/qubesattachment.js:35]: Does it work also on older thunderbird versions (as you noted in manifest: >=14)?

Comment (m-v-b) [reply chrome/content/qubesattachment.js:35]: It appears to work with Thunderbird 14.0, so I am assuming that all intermediate versions would work too.

More details: I created a temporary appVM, and I downloaded Thunderbird 14.0 from [here](https://ftp.mozilla.org/pub/thunderbird/releases/14.0/linux-x86_64/en-US/). Afterwards, I transferred to the appVM an e-mail file (`*.eml`) containing an attachment. I tested the open-attachment-in-disposable-VM feature, and it worked.

Here is a screenshot of the add-on's preferences window along with Thunderbird's version window. Perhaps the screenshot is not sufficient on its own to demonstrate that the changes work though, given that I did not up-version the add-on.

screenshot

**Comment by m-v-b** [comment]
For the record, the automated check failure encountered by Travis is due to a connectivity issue (Debian mirror appears to time out). Is it possible for me to re-trigger the check procedure?

**Comment by marmarek** [comment]
> Is it possible for me to re-trigger the check procedure?

No, but I can do that (done).

**Review by marmarek — APPROVED** [review]

**Comment by marmarek** [comment]
Thanks!

**Pull request merged**